identity-apps-core/apps/authentication-portal/src/main/webapp/login.jsp

```diff
@@ -1265,6 +1265,12 @@
     <script src="util/string-utils.js"></script>
 
     <script>
+
+        <% if (Boolean.parseBoolean(request.getParameter("isSelfRegistration"))) { %>
+                $(".ui.segment").hide();
+                window.location = "<%=getRegistrationUrl(accountRegistrationEndpointContextURL, srURLEncodedURL, (String) request.getAttribute(JAVAX_SERVLET_FORWARD_QUERY_STRING))%>";
```

Comment:
check if similar logic as below works here
request.getRequestDispatcher("error.jsp").forward(request, response);

Reply:
> check if similar logic as below works here request.getRequestDispatcher("error.jsp").forward(request, response);

This is not possible since the registration page resides in a seperate web app (recovery portal)

Reply:
> Have your tried using `response.sendRedirect` from Java context?

Yes, since the login page url is generated and added to the auth response as a 302 location header, using sendRedirect in this flow will introduce some inconsistencies and unwanted complexity.